### DISCUSSION TITLE
Google Meet, Discord etc. not seeing Blackhole on Safari

### DISCUSSION THREAD

**Discussion by ghost** [ExistentialAudio/BlackHole#514] [Q&A]
Hi,

My issue: various web-based conferencing tools like Google Meet or Discord won't show any "blackhole" option among the possible audio inputs in their settings anymore on Safari. That option was definitely there previously, I used to use my daw to capture my mic's audio on its own, dedicated track, apply eq and compression to it, have some other synths and samplers on other tracks and have "blackhole" as my output device there, so I could stream all of that directly into Google Meet for online audio lessons. Well, a multi-output device actually, mixing "blackhole" with my speakers so I could also monitor everything, but Google Meet's mic input would be set to "blackhole" anyway.

Could this be related to a system software update? Using a 2020 M1 MacBook Air, came with Monterrey 12 which I recently updated to 12.1. Or maybe it was 12.1 right from the start? Anyway, I downloaded and installed a system software update no longer than 12 days ago and after that, the "blackhole" input option was gone... Anybody else having this issue?

**Comment by devinroth** [comment]
That’s a Safari issue. It seems to be really common and not just related to BlackHole.

I think there’s a security setting for that somewhere but it’s beyond the scope for me.

**Reply by ghost** [reply]
Thanks! Gonna have a look
